### PR TITLE
pycon 2017 location fix

### PR DESCRIPTION
**Pull request by juggernaut451**
fixes: https://github.com/pythonindia/inpycon2018/issues/39

### CODE REVIEW

File: js/content.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ var contentArray = [
     "taskTitle": "PyCon India 2017",
     "title": "9th Year of PyCon India",
     "date": "2nd - 5th November, 2017",
-    "location": "Indian Institute of Science, Bangalore",
+    "location": "Shaheed Sukhdev College Of Business Studies, New Delhi",
     "speakers": "&#9679; Noufal Ibrahim <br>&#9679; Elizabeth Ferrao <br>&#9679; Peter Wang",
     "description": "PyCon India, the premier conference in India on using and \
                     developing the Python programming language is conducted \
```
Comment (satyaakam) [inline js/content.js]: @realslimshanky why do we need to make this change happen ?

Comment (realslimshanky) [reply js/content.js]: The location is incorrect for 2017 PyCon India conference as stated in this issue #39

**Comment by realslimshanky** [comment]
@chirag200666 need to look into this as well.

**Comment by chirag200666** [comment]
@juggernaut451  @realslimshanky   @ananyo2012
Guys thanks for the good work, merged it.

**Pull request merged**